src/components/cards/ctaCard.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import SbEditable from "storyblok-react";
+import Icon from "react-hero-icon";
+import { Heading, SrOnlyText } from "decanter-react";
+import { dcnb } from "cnbuilder";
+import { heroicon, ctaLinkIconColor } from "../../utilities/dataSource";
+import SbLink from "../../utilities/sbLink";
+
+const CtaCard = React.forwardRef(
+  (
+    {
+      blok: { headline, headingLevel, icon, linkText, link, rel, srText },
+      blok,
+    },
+    ref
+  ) => {
+    let iconClasses;
+
+    if (icon === "external") {
+      iconClasses =
+        "su-h-08em su-w-08em su-ml-4 su--mt-2 su-transform-gpu su-rotate-45 group-hocus:su-rotate-45";
+    } else {
+      iconClasses = "su-h-1em su-w-1em su-ml-04em su--mt-2";
+    }
+
+    // Icon color
+    const iconColor = ctaLinkIconColor["white-hover-digital-red"];
+
+    // Icon animation
+    let iconAnimate = "su-transition-transform group-hocus:su-transform-gpu";
+
+    if (icon === "external") {
+      iconAnimate = dcnb(
+        iconAnimate,
+        "group-hocus:su-translate-x-01em group-hocus:su--translate-y-01em"
+      );
+    } else {
+      iconAnimate = dcnb(iconAnimate, "group-hocus:su-translate-x-02em");
+    }
+
+    // Heroicon option
+    const linkIcon = heroicon[icon] ?? heroicon["arrow-right"];
+
+    return (
+      <SbEditable content={blok}>
+        <div
+          className={dcnb(
+            "cta-card su-w-full su-relative su-bg-digital-red su-text-white hocus:su-bg-cardinal-red-xdark su-basefont-23 su-break-words su-p-50 su-flex su-flex-col su-justify-end"
```

Comment:
```suggestion
            "cta-card su-w-full su-relative su-bg-digital-red su-text-white hocus:su-bg-cardinal-red-xdark su-basefont-23 su-break-words su-rs-py-3 su-rs-px-2 su-flex su-flex-col su-justify-end"
```
Let's use responsive-spacing for card padding 😄